### DISCUSSION TITLE
Imagemagick upgrade to ImageMagick-7.1.0.62 . png and jpeg conversion to webp has very large size compared to previous size

### DISCUSSION THREAD

**Discussion by sunnychampion123** [ImageMagick/ImageMagick#6652] [General]
Imagemagick upgrade to ImageMagick-7.1.0.62 . png and jpeg conversion to webp has very large size compared to previous size

**Comment by fmw42** [comment]
There is no 7.1.0.62. the current version is 7.1.0.16

**Reply by sunnychampion123** [reply]
Please check the below. There is 7.1.0.62

https://github.com/ImageMagick/ImageMagick/compare/7.1.0-61...7.1.0-62

**Reply by fmw42** [reply]
Releases versions are at https://imagemagick.org/archive/

**Reply by dlemstra** [reply]
Fred, they are using an older version. and the release you mean is probably `7.1.1.16` instead.

**Reply by fmw42** [reply]
OOPS!  7.1.1.16 not 7.1.0.16.  Sorry, I overlooked the third digit as 0 in his version.

**Reply by sunnychampion123** [reply]
is there any update/recompile is required to libwebp??

**Reply by sunnychampion123** [reply]
png and jpeg conversion to webp has very large size compared to previous size

**Reply by sunnychampion123** [reply]
we have a prod environment in which we are running 7.0.10-7. Compared to prod stage convert webp image are very large
stage is running on 7.1.0.62
Prod is running on 7.0.10-7

**Comment by snibgo** [comment]
You don't provide any detail of the problem. What commands do use use? What defines from the page https://imagemagick.org/script/webp.php do you use? How large are the output files? What are your input files?
